Tests/SonarScanner.MSBuild.PreProcessor.Test/ProcessedArgsTests.cs

```diff
@@ -319,23 +339,32 @@ public void ProcArgs_HostUrl_SonarcloudUrl_PropertyAggregation()
         }
 
         [DataTestMethod]
-        [DataRow(true, true, true, 3)]
-        [DataRow(false, true, true, 2)]
-        [DataRow(true, false, true, 2)]
-        [DataRow(true, true, false, 2)]
-        [DataRow(false, false, true, 1)]
-        [DataRow(false, true, false, 1)]
-        [DataRow(true, false, false, 1)]
-        [DataRow(false, false, false, 0)]
-        public void ProcArgs_ErrorAndIsValid(bool invalidKey, bool invalidOrganization, bool invalidHost, int errors)
+        [DataRow(true, true, true, true, 4)]
+        [DataRow(false, true, true, true, 3)]
+        [DataRow(true, false, true, true, 3)]
+        [DataRow(true, true, false, true, 3)]
+        [DataRow(false, false, true, true, 2)]
+        [DataRow(false, true, false, true, 2)]
+        [DataRow(true, false, false, true, 2)]
+        [DataRow(false, false, false, true, 1)]
+        [DataRow(true, true, true, false, 3)]
+        [DataRow(false, true, true, false, 2)]
+        [DataRow(true, false, true, false, 2)]
+        [DataRow(true, true, false, false, 2)]
+        [DataRow(false, false, true, false, 1)]
+        [DataRow(false, true, false, false, 1)]
+        [DataRow(true, false, false, false, 1)]
+        [DataRow(false, false, false, false, 0)]
```

Comment:
```suggestion
        [DataRow(true, true, true, true, 4)]
        [DataRow(true, true, true, false, 3)]
        [DataRow(true, true, false, true, 3)]
        [DataRow(true, true, false, false, 2)]
        [DataRow(true, false, true, true, 3)]
        [DataRow(true, false, true, false, 2)]
        [DataRow(true, false, false, true, 2)]
        [DataRow(true, false, false, false, 1)]
        [DataRow(false, true, true, true, 3)]
        [DataRow(false, true, true, false, 2)]
        [DataRow(false, true, false, true, 2)]
        [DataRow(false, true, false, false, 1)]
        [DataRow(false, false, true, true, 2)]
        [DataRow(false, false, true, false, 1)]
        [DataRow(false, false, false, true, 1)]
        [DataRow(false, false, false, false, 0)]
```

I've sorted this alphabetically (reversed) since I find it easier to follow. Up to you if you.

Reply:
I think this is up to you, as the PR author to decide. 

What I would prefer though, is to add  CombinatorialDataTestMethodAttribute in dedicated PR.